draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>When an RTP packet is received by a transport (typically a
```

Comment:
I think right here we should put something like "Note: The algorithm described in this section does not yet have WG consensus but is included here so there is something concrete for the WG to decide if they are OK with." so that we don't get howls from some people.